config/crowdsec-cloudflare-worker-bouncer_docker.yaml

```diff
@@ -0,0 +1,43 @@
+crowdsec_config:
+  lapi_url: ${CROWDSEC_LAPI_URL}
+  lapi_key: ${API_KEY}
+  update_frequency: 10s
+  include_scenarios_containing: []
+  exclude_scenarios_containing: []
+  only_include_decisions_from: ["cscli", "crowdsec"]
```

Comment:
Can this be empty, as users may not know this and wonder why they stop getting decisions from CAPI